src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the path to `../images/star.svg` is correct and that the image file exists at this location.